migrator/migrations/n_39_to_n_40_postgres_policies/legacy/files/add_instruction.json

```diff
@@ -1,7 +1,7 @@
 {
   "id": "80267b36-2182-4fb3-8b53-e80c031f4ad8",
   "name": "ADD Command used instead of COPY",
-  "description": "Alert on deployments using a ADD command",
+  "description": "Alert on deployments using an ADD command",
```

Comment:
Please don't change this file. Nothing in existing migration files should be modified